app/core/models_core.py

```diff
@@ -35,9 +35,16 @@ class CoreUser(Base):
     birthday: Mapped[date | None] = mapped_column(Date)
     promo: Mapped[int | None] = mapped_column(Integer)
     phone: Mapped[str | None] = mapped_column(String)
-    floor: Mapped[FloorsType] = mapped_column(Enum(FloorsType), nullable=False)
+    floor: Mapped[FloorsType | None] = mapped_column(Enum(FloorsType))
```

Comment:
I don't know if we want to make this column nullable since Titan always sends a value

Reply:
It was done, because we now allow external account to be created, and they can be created via raid-registering, for which it makes no sense to specify a floor